03_letters/CameronSchafer/main_test.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+// main() test
+func TestMainOutput(t *testing.T) {
```

Comment:
Again here, I don't see these comments adding any value. The name of the test is documentation in itself. Please remove for all these

Reply:
Done